src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $day-block-size * 7 + $gap-size * 6 + $side-padding * 2;
+  padding: 25px $side-padding $side-padding;
+  gap: $gap-size;
+
+  &__day {
+    box-sizing: border-box;
+    width: $day-block-size;
+    height: $day-block-size;
+    border: 1px solid #000;
+
+    background-color: #eee;
+    text-align: center;
+    transition: 0.5s;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      transform: translatey(-20px);
+    }
+  }
+
+  @for $number from 1 through 31 {
+    & :nth-child(#{$number})::before {
+      content: "#{$number}";
+      line-height: $day-block-size;
+      color: #000;
+    }
+  }
+
+  @for $number from 28 through 31 {
+    &--month-length--#{$number} {
+      & :nth-child(n + #{$number + 1}) {
+        visibility: hidden;
```

Comment:
Not critical, but I think `display: none` is better to completely remove elements from page